cwl_adapters/onionnet-sfct.cwl

```diff
@@ -9,7 +9,7 @@ baseCommand: ["python", "/OnionNet-SFCT/scorer.py"]
 
 hints:
   DockerRequirement:
-    dockerPull: cyangnyu/onionnet-sfct
+    dockerPull: polusai/onionnet-tool:1.0.0
```

Comment:
Onionnet sfct is completely different from 'regular' onionnet (v1 and/or v2), so we should keep sfct in the name.